docs/bidders/tappx.md

```diff
@@ -0,0 +1,13 @@
+# Tappx Bidder
+
+## Parameters
+Please contact [Tappx](../../static/bidder-info/tappx.yaml) to get set up. Our operations team will provide the 3 required parameters:
```

Comment:
Are we sure we want a link to the `yaml` file and not an actual "contact us" page?